Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:4.0
```

Comment:
Is that all that is needed? No need to update any of the dependencies ? If so, nice !

Reply:
Yep!